src/app/component/home/models/NewsDto.ts

```diff
@@ -1,12 +1,6 @@
-export class NewsDto {
+export interface NewsDto {
   id: number;
   title: string;
   text: string;
   creationDate: string;
```

Comment:
explain those changes

Reply:
![image](https://user-images.githubusercontent.com/33227414/90184372-ae3cd080-ddbd-11ea-98d1-26dbb2062369.png)

Reply:
🙂 the question is - why you need this change?

Reply:
I know, that functionality isn't for aot-mode, but it looks like more readable.

Reply:
fixed